dyn_nmm/module_PHYSICS_CALLS.F

```diff
@@ -197,6 +197,8 @@ SUBROUTINE RADIATION(NTSD,DT,JULDAY,JULYR,XTIME,JULIAN            &
                                                         SWNIRDIR, &
                                                         SWNIRDIF
 
+      REAL,DIMENSION(IMS:IME,KMS:KME,JMS:JME) :: RTHRATENLWC
```

Comment:
@jhruppert 
Are you able to add in the INTENT for these clear sky variables?

Reply:
@davegill 
Dave, no problem, just added these.